packages/neo4j-driver-deno/lib/core/internal/connection-holder.ts

```diff
@@ -161,9 +166,9 @@ class ConnectionHolder implements ConnectionHolderInterface {
     return this._referenceCount
   }
 
-  initializeConnection (): boolean {
+  initializeConnection (homeDatabaseTable?: any): boolean {
```

Comment:
```suggestion
  initializeConnection (homeDatabaseTable?: string): boolean {
```
At least that's what `_createConnectionPromise` expects and I don't see any narrowing type checks or such here.